Find a functionally equivalent code:com.android.framework.permission.tests.VibratorServicePermissionTest.testVibratePattern:COMMENT
Method Modifier: public      
Comment:/**
 * Test that calling {@link android.os.IVibratorService#vibratePattern(long[],
 * int, android.os.IBinder)} requires permissions.
 * <p>Tests permission:
 * {@link android.Manifest.permission#VIBRATE}
 * @throws RemoteException
 */

@@ -1,6 +1,6 @@
 {
     try {
-        mVibratorService.vibratePattern(Process.myUid(), null, new long[] { 0 }, 0, new Binder());
+        mVibratorService.vibratePattern(Process.myUid(), null, new long[] { 0 }, 0, AudioManager.STREAM_ALARM, new Binder());
         fail("vibratePattern did not throw SecurityException as expected");
     } catch (SecurityException e) {
     // expected

